spot-client/src/spot-tv/ui/components/meeting-frame/ZoomMeetingFrame/ZoomMeetingFrame.js

```diff
@@ -185,14 +217,16 @@ export class ZoomMeetingFrame extends AbstractMeetingFrame {
                 });
             } else if (zoomErrorCode === errorCodes.MEETING_NOT_START) {
                 logger.log('Zoom meeting has not started');
-                this.props.onMeetingLeave({
-                    errorCode: 'meeting-not-started',
-                    error: 'appEvents.meetingNotStarted'
-                });
+
+                this.props.updateSpotTvState({ waitingForMeetingStart: true });
+                this._enqueueJoinRetry();
             } else if (zoomErrorCode === errorCodes.WRONG_MEETING_PASSWORD) {
                 logger.log('password required');
 
-                this.props.updateSpotTvState({ needPassword: true });
+                this.props.updateSpotTvState({
+                    needPassword: true,
+                    waitingForMeetingStart: true
```

Comment:
why does the password case also have `waitingForMeetingStart` set to `true`?

Reply:
Oh good point. It should be false. Makes me wonder what I was testing when I made this change... Probably the waitingForMeetingStart  state got cleared on meeting join so I wasn't testing anything.

Reply:
Yes, I definitely wanted this to be false. it hides the waiting overlay.